src/auto/dbus_arg_info.rs

```diff
@@ -0,0 +1,17 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use gio_sys;
+use glib::translate::*;
+
+glib_wrapper! {
+    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct DBusArgInfo(Shared<gio_sys::GDBusArgInfo>);
```

Comment:
This is not useful as it is. Needs some accessors for the struct fields manually implemented, same for `DbusPropertyInfo` and `DbusMethodInfo` and all those other `Info` structs

Reply:
The main use of `Info` structs is to do `gio::DBusNodeInfo::new_for_xml`, `.lookup_interface` and pass the `DBusInterfaceInfo` to `register_object_with_closures` :)

Actual inspection of info would be nice I guess, but not a priority for me.

Reply:
Doesn't look like much is missing for that though, just the field accessors or not?

Reply:
(should create an issue about that so it can be added later and is not forgotten if you don't want to add it)

Reply:
Nothing is missing, you can see I'm using it in the example in the first post here.

Reply:
No I mean inspection is missing, and the struct accessors for these `Info` structs seem to be the only thing missing for that :)

You're right that it can actually also be used otherwise like you did via the XML, I hadn't thought of that.

Reply:
I'll add these once this one is merged here